### PR TITLE
Add simple heapster and prometheus examples as optional templates to kube-system

### PR DESCRIPTION
**Pull request by smarterclayton**
Gives admins simple Kube equivalent capabilities in Origin

### CODE REVIEW

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
@csrwng

File: examples/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -72,7 +73,7 @@ objects:
           imagePullPolicy: IfNotPresent
           name: prometheus
           ports:
-          - containerPort: 9090
```
Comment (csrwng) [inline examples/prometheus/prometheus.yaml]: The process inside the pod still listens on 9090. There's likely something you need to change in the config to have it listen on 8080.

**Comment by csrwng** [comment]
The cluster up parts LGTM
Just one comment on the prometheus template.

And an observation... as an admin user I was not able to instantiate the heapster template through the web console because the web console doesn't know about clusterrolebindings. /cc @jwforres

**Comment by smarterclayton** [comment]
Fixed to go back to 9090, [merge]

**Pull request merged**